### ISSUE TITLE
[Bug]: Opened apps don't show in taskbar

### ISSUE THREAD

**Issue by jin-li** [prasanthrangan/hyprdots#1582]
### Description

The opened apps don't show up in `wlr/taskbar` component in `waybar`.

### Steps to Reproduce

1. Open any app(s)

### Expected Behavior

The icons of the opened apps should appear in the `taskbar` region in the `waybar`.

### Actual Behavior

The `taskbar` region only shows the background, no apps appear in that region. See the picture in the screenshots.

### Screenshots

![240613_03h37m54s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/26253144/36bdb136-dcdc-4522-b121-6bb4f3daa381)

### Environment

- Linux Distro: Arch Linux
- Linux Version, output of `uname -a`: Linux archlinux 6.9.3-arch1-2-surface
- Hyprland Version, output of `hyprctl version`: v0.41.0
- Installation Environment: On the main hardware,  Surface Pro 6
- Other info: see the `fastfetch` output below

![240613_03h34m33s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/26253144/7af41068-6b0d-4dcd-99c5-2e5056daa79e)

### Additional Information

Here are the debug outputs when I run `pkill waybar; waybar`:

```sh
❯ pkill waybar
❯ waybar
[2024-06-13 03:18:50.015] [info] Using configuration file /home/lijin/.config/waybar/config.jsonc
[2024-06-13 03:18:50.017] [info] Unable to receive desktop appearance: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such interface “org.freedesktop.portal.Settings” on object at path /org/freedesktop/portal/desktop
[2024-06-13 03:18:50.017] [info] Using CSS file /home/lijin/.config/waybar/style.css
[2024-06-13 03:18:50.036] [info] Hyprland IPC starting
[2024-06-13 03:18:50.038] [info] Loading persistent workspaces from Waybar config
[2024-06-13 03:18:50.038] [info] Loading persistent workspaces from Hyprland workspace rules

** (waybar:1553): WARNING **: 03:18:50.202: Status Notifier Item with bus name ':1.19' and object path '/org/ayatana/NotificationItem/nm_applet' is already registered

** (waybar:1553): WARNING **: 03:18:50.202: Status Notifier Item with bus name ':1.28' and object path '/org/ayatana/NotificationItem/udiskie' is already registered
[2024-06-13 03:18:50.220] [info] Bar configured (width: 1368, height: 31) for output: eDP-1

** (waybar:1553): WARNING **: 03:18:52.102: Status Notifier Item with bus name ':1.19' and object path '/org/ayatana/NotificationItem/nm_applet' is already registered

** (waybar:1553): WARNING **: 03:18:52.102: Status Notifier Item with bus name ':1.28' and object path '/org/ayatana/NotificationItem/udiskie' is already registered
```
These are the only outputs, I didn't see more outputs when I open apps/windows/workspaces.

Any clue what's the problem? Or how to debug it? Thanks very much!

**Comment by jin-li** [comment]
Sorry for the repetition. This seems like the same bug as #1579

**Comment by jin-li** [comment]
This has been fixed by the hyprland upstream: [Hyprland source changes frequently, it is recommended to use the -git version of the dependent hyprwm libraries](https://aur.archlinux.org/packages/hyprland-git). 

TL;DR:
```sh
yay -Sy hyprutils-git hyprcursor-git hyprlang-git hyprwayland-scanner-git 
yay -Sy aur/hyprland-git
```
